.github/workflows/run-tests.yml

```diff
@@ -120,6 +120,7 @@ jobs:
       run: picard --long-version --no-crash-dialog
     - name: Verify sdist package
       if: runner.os != 'Windows'
+      timeout-minutes: 40
```

Comment:
Any specific reason for the 40 here?

Reply:
Let's make it 30 anyway, keep it consistent. It should still be under 10 minutes (and that's long already), so 30 is fine